src/encoding/src/tests/base64_felt_test.cairo

```diff
@@ -0,0 +1,78 @@
+use alexandria_data_structures::array_ext::ArrayTraitExt;
+use alexandria_encoding::base64::{
+    Base64Encoder, Base64UrlEncoder, Base64FeltEncoder, Base64UrlFeltEncoder
+};
+use core::debug::PrintTrait;
+use core::traits::TryInto;
+
+fn bytes_be(val: felt252) -> Array<u8> {
+    let mut result = array![];
+
+    let mut num: u256 = val.into();
+    loop {
+        if num == 0 {
+            break;
+        }
+        let (quotient, remainder) = DivRem::div_rem(
+            num, 256_u256.try_into().expect('Division by 0')
+        );
+        result.append(remainder.try_into().unwrap());
+        num = quotient;
+    };
+    loop {
+        if result.len() >= 32 {
+            break;
+        }
+        result.append(0_u8);
+    };
+    result = result.reverse();
```

Comment:
same why not backward to avoid a reverse ?

Reply:
I've thought the additional division required to make this work would outweigh any potential benefit. Upon your suggestion I've tested my hypothesis which turned out to be correct. The code below is around 50% slower compared to one with the ```.reverse()```. 
```cairo
const TWO_TO_248: felt252 = 452312848583266388373324160190187140051835877600158453279131187530910662656;

fn bytes_be_rev(val: felt252) -> Array<u8> {
    let mut result = array![];

    let mut num: u256 = val.into();
    let mut divider: u256 = TWO_TO_248.into();
    loop {
        if num == 0 || divider == 0 {
            break;
        }
        let (quotient, remainder) = DivRem::div_rem(num, divider.try_into().unwrap());
        result.append(quotient.try_into().unwrap());
        num = remainder;
        divider = divider / 256_u256;
    };
    loop {
        if result.len() >= 32 {
            break;
        }
        result.append(0_u8);
    };
    result
}
```
There reason for that is in my opinion the additional assignment of ```divider```` which cancels out with the ```.reverse()``` and the aforementioned division which worsens the performance.

Did you have something other in mind or do you see some potential improvements?

Reply:
no looks good will merge it as is